src/openzeppelin/tests/security/test_pausable.cairo

```diff
@@ -92,8 +111,15 @@ fn test_pause_when_paused() {
 #[available_gas(2000000)]
 fn test_unpause_when_paused() {
     let mut state = STATE();
+    testing::set_caller_address(CALLER());
+
     InternalImpl::_pause(ref state);
+    testing::pop_log_raw(ZERO());
+
```

Comment:
Since the `Paused` event was included in the `pop_log_raw` even though it's called after `_pause`, should we just move `pop_log_raw` to the beginning of the function body with the other context setups? It seems like it'll capture every event regardless of where it's called